trillian/integration/ct_functions.sh

```diff
@@ -63,17 +62,9 @@ ct_prep_test() {
     if [[ ! -z "${ETCD_OPTS}" ]]; then
         PROMETHEUS_CFGDIR="$(mktemp -d ${TMPDIR}/ct-prometheus-XXXXXX)"
         local prom_cfg="${PROMETHEUS_CFGDIR}/config.yaml"
-        local etcdiscovered="${PROMETHEUS_CFGDIR}/trillian.json"
-        sed "s!@ETCDISCOVERED@!${etcdiscovered}!" ${CT_GO_PATH}/trillian/integration/prometheus.yml > "${prom_cfg}"
         echo "Prometheus configuration in ${prom_cfg}:"
         cat ${prom_cfg}
 
-        echo "Building etcdiscover"
-        go build github.com/google/trillian/monitoring/prometheus/etcdiscover
-
-        echo "Launching etcd service monitor updating ${etcdiscovered}"
-        ./etcdiscover ${ETCD_OPTS} --etcd_services=trillian-ctfe-metrics-http,trillian-logserver-http,trillian-logsigner-http -target=${etcdiscovered} --logtostderr &
-        ETCDISCOVER_PID=$!
         echo "Launching Prometheus (default location localhost:9090)"
         ${PROMETHEUS_DIR}/prometheus --config.file=${prom_cfg} \
```

Comment:
If the content in `${prom_cfg}` is always empty, would there be an issue when the Prometheus is launched?

Reply:
Good spot. This is part of the problem from trying to fix up code that has bit-rotted. I've pointed this at the static config file now.